webapp/src/components/workspaceSwitcher/workspaceOptions.tsx

```diff
@@ -102,7 +103,13 @@ const Option = (props: any): JSX.Element => {
             {
                 props.data.value !== DashboardOption.value &&
                 <div className='boardCount'>
-                    {props.data.boardCount} {props.data.boardCount > 1 ? 'Boards' : 'Board'}
+                    {
+                        intl.formatMessage({
+                            id: 'General.BoardCount',
+                            defaultMessage: `${props.data.boardCount} Board${props.data.boardCount === 1 ? '' : 's'}`,
```

Comment:
The defaultMessage should be the same used in the `i18n/es.json`

Reply:
Done.